app/view_objects/assessor_interface/application_forms_show_view_object.rb

```diff
@@ -128,10 +127,12 @@ def initial_assessment_task_list_section
         I18n.t(
           "assessor_interface.application_forms.show.assessment_tasks.sections.initial_assessment",
         ),
-      items: [
-        *assessment_section_task_list_items(preliminary: false),
-        initial_assessment_recommendation_task_list_item,
-      ],
+      items:
+        (
+          assessment_section_task_list_items(preliminary: false) +
+            [initial_assessment_recommendation_task_list_item] +
+            further_information_task_list_section[:items]
```

Comment:
Since it's only being used here to get the items I think we can change the `further_information_task_list_section` method to return the items directly (and probably rename the method accordingly).